.env

```diff
@@ -0,0 +1,6 @@
+DB_HOST=https://nodejs-lists-app.vercel.app/
+
+NEXT_PUBLIC_API_URL=https://nodejs-lists-app.vercel.app/
+
+NEXT_PUBLIC_TOKEN=
```

Comment:
Does this work? One of my projects I had to use the token without quotes. Just NEXT_PUBLIC_TOKEN=token_example

Reply:
it wasn't work without quotes for me)) It is interesting. I will recheck and tell you.

Reply:
Yeap, it works only with quotes. Other way I get an error with 'jwt must be provided' message